hangman.js

```diff
@@ -1 +1,28 @@
-console.log("welcome to hangman");
\ No newline at end of file
+function playHangman(){
+  console.log("welcome to hangman");
+  words = ['javascript','ruby','rails','jquery','programming'];
+  dashedWords = ['- - - - - - - - - -','- - - -','- - - - -','- - - - - -','- - - - - - - - - - -'];
+  guesses = [];
+  incorrectGuesses = 0;
+function randomizeIndex(){
+    randomIndex = Math.floor(Math.random()*5);
```

Comment:
how could you avoid setting global variables? Could this function return a value?